kura/org.eclipse.kura.driver.opcua.provider/src/main/java/org/eclipse/kura/internal/driver/opcua/OpcUaDriver.java

```diff
@@ -106,79 +102,24 @@
     /** Node Namespace Index Property */
     private static final String NODE_NAMESPACE_INDEX = "node.namespace.index";
 
-    /** URL Converter Service Trackers */
-    private static Map<String, ServiceTracker<Object, URLConverter>> urlConverterTrackers = CollectionUtil
-            .newConcurrentHashMap();
-
-    // initialize Milo Stack class loader
-    static {
```

Comment:
Yes, the class loading issues were caused by some missing packages that were used by Milo deps. I've imported those packages and the issues should be solved.